docs/cse/automation-service/about-automation-service.md

```diff
@@ -68,9 +68,18 @@ An automation runs a playbook, which  runs actions that are provided by integrat
 Before you can access the Automation Service, you must first [configure role capabilities](#configure-role-capabilities).
 :::
 
-1. Click the **Configuration** button (gear icon) at the top of the Cloud SIEM UI.
-1. Under **Integrations**, select **Automation**.<br/><img src={useBaseUrl('img/cse/automations-config-menu.png')} alt="Automation menu option" width="150"/><br/>The list of available automations appears. Each automation runs a playbook.<br/><img src={useBaseUrl('img/cse/automations-automations-list.png')} alt="Automations list" width="800"/>
-1. To [view playbooks](/docs/cse/automation-service/automation-service-playbooks#view-playbooks), at the top of the screen click **Manage Playbooks**.<br/><img src={useBaseUrl('img/cse/automations-manage-playbooks.png')} alt="Manage Playbooks menu option" width="400"/>.<br/>The list of available playbooks displays. Playbooks run actions provided by integrations.<br/><img src={useBaseUrl('img/cse/automations-playbook-list.png')} alt="Automation Playbook list" width="800"/>
+1. To access the Automation Service:
+   * From the Sumo Logic screen:
+     1. Go to the main menu.
+     1. Click **Automation**. <br/><img src={useBaseUrl('img/cse/automation-menu-in-nav-bar.png')} alt="Automation menu option in the nav bar" width="200"/> 
+     :::note
+     The **Automation** option appears in the main menu only if you have Cloud SIEM installed. If you also have Cloud SOAR installed, a **Cloud SOAR**  option appears instead, since all automation services are provided by Cloud SOAR when it installed in conjunction with Cloud SIEM.
```

Comment:
```suggestion
     The **Automation** option appears in the main menu only if you have Cloud SIEM installed. If you also have Cloud SOAR installed, a **Cloud SOAR** option appears instead, since all automation services are provided by Cloud SOAR when it installed in conjunction with Cloud SIEM.
```